test/test_librosa_compatibility.py

```diff
@@ -115,12 +117,8 @@ def test_amplitude_to_DB(self):
 ])
 @pytest.mark.parametrize('rate', [0.5, 1.01, 1.3])
 @pytest.mark.parametrize('hop_length', [256])
+@unittest.skipIf(not LIBROSA_AVAILABLE, "Librosa not available")
```

Comment:
Thanks for addressing [comment](https://github.com/pytorch/audio/issues/477#issuecomment-613528225)